source/qml_v2/AdaptiveCardQmlEngine/qml/RichTextBlockRender_old.qml

```diff
@@ -0,0 +1,36 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.3
+import QtQuick.Controls 2.15
+import AdaptiveCardQmlEngine 1.0
+import "JSUtils/AdaptiveCardUtils.js" as AdaptiveCardUtils
+
+TextEdit {
```

Comment:
What is this `_old` file?

Reply:
This was a duplicate file of RichTextBlock.qml which was created by mistake and is not required so i have removed this in the latest commit changes.